workshop_operations/onboarding-checklist.md

```diff
@@ -4,61 +4,84 @@
+- [ ] Create a GitHub profile for yourself
+- [ ] Ask carpadmin @ swcarpentry dot uio dot no to invite you as a member of uio-carpentry https://github.com/uio-carpentry
+- [ ] Open https://github.com/carpentries/workshop-template and click the ```Use this template``` button in green
+- [ ] Choose "uio-carpentry" as owner, set the name of the repository YYYY-MM-DD-lesson, make sure that "Public" is chosen and click "Create repository from template" button in green.
 - [ ] Edit the necessary parts by following README.md "Customizing Your Website"
 - [ ] The website will be automatically created
-- [ ] Paste the link into the planning spreadsheet
+- [ ] Send the link to the website to carpadmin @ swcarpentry dot uio dot no – the coordinators will create an event on the University Library's course catalogue and create a registration form
+- [ ] Submit your workshop to the Carpentries database by following the instructions on this website https://amy.carpentries.org/forms/self-organised/
 ---
 
-* **How do I get in contact with workshop participants?**
-- [ ] Download registration list from Nettskjema
-- [ ] Send email to those who registered, using BCC
-- [ ] Send reminder before workshop
+### **How do I get in contact with workshop participants?**
+- [ ] Registration for workshops is handled by the local coordinator
+- [ ] By default, registration is closed two weeks prior to the workshop date
+- [ ] Automatic reminders for workshop participants are sent out via the registration system one week and again three days before the workshop
+- [ ] If you want to get in contact with participants, the local coordinator will send you the participants contacts
+- [ ] Workshop participants can ask for a Certificate of Attendance after the workshop; the certificate is generated automatically
 ---
 
-* **Which room(s) will I be teaching in?**
+### **Which room(s) will I be teaching in?**
 <!-- Update this list for each new onboarding session -->
+- [ ] Seminar room 123 in Vilhelm Bjerknes' hus (the Science Library, ground floor)
 - [ ] Teaching room 209 in the Science Library - Science Fiction Section
-- [ ] Abels Utsikt on the top floor of the Mathematics Building - Niels Henrik Abels hus
+- [ ] You can ask the local coordinator to book a room for you
+
+<!-- - [ ] Abels Utsikt on the top floor of the Mathematics Building - Niels Henrik Abels hus
 - [ ] Seminar room 3508 in the Biosciences Building - Kristine Bonnevies hus
-- [ ] Group room 4436 in the Biosciences Building - Kristine Bonnevies hus
-- [ ] You can ask the library contact to book a room for you
+- [ ] Group room 4436 in the Biosciences Building - Kristine Bonnevies hus -->
 ---
 
-* **Where do I find the Carpentries equipment (cables, extension cords, sticky notes)?**
+### **Where do I find the Carpentries equipment (cables, extension cords, sticky notes)?**
 ---
 
-* **What do I have to do before, during, and after a workshop?**
+### **What do I have to do before, during, and after a workshop?**
```

Comment:
Should we merge the contents of [Carpentries handbook instructor checklist](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html#instructor-checklist) so that it will be easier for the instructors to read the contents at one place? The same applies to helpers ([helpers checklist](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html#helper-checklist))